server/builtin/install.go

```diff
@@ -11,7 +11,8 @@ import (
 )
 
 func (a *builtinApp) installCommandBinding(loc *i18n.Localizer) apps.Binding {
-	if a.conf.Get().MattermostCloudMode {
+	conf := a.conf.Get()
+	if conf.MattermostCloudMode && !conf.DeveloperMode {
```

Comment:
Why should this command _not_ work in dev mode?

Reply:
This is the "cloud-only" mode of the command, no sub-commands.